src/pages/home/HomePage.js

```diff
@@ -6,13 +6,14 @@ import {
     Animated,
     Easing,
     Keyboard,
-    Pressable,
+    Pressable
```

Comment:
small detail - we always include trailing commas where valid. This is part of our style guide, but if it is not reflected by our ESLint rules we should try to fix that.

Reply:
Created issue for this here: https://github.com/Expensify/Expensify/issues/149302
Has bothered me for a while 😄

Reply:
> small detail - we always include trailing commas where valid. This is part of our style guide, but if it is not reflected by our ESLint rules we should try to fix that.

Done.

Reply:
> Created issue for this here: [Expensify/Expensify#149302](https://github.com/Expensify/Expensify/issues/149302)
> Has bothered me for a while 😄

Expensify/Expensify#149302 shows 404

Reply:
> Expensify/Expensify#149302 shows 404

No need to worry about this! Thanks for adding the trailing comma 👍